unit-3-js-labs/counting-loops/index.js

```diff
@@ -0,0 +1,48 @@
+for (let i = 1; i<=10; i++) {
+    console.log (i)    
+    for (let j = 10; j >= 1; j--) {  
+        console.log(j)
+    }
+}
```

Comment:
> Use a for loop to log numbers from 1 to 10, then another for loop to count down, logging numbers from 10 to 1.

This means there's two separate loops, not one nested loop.

```suggestion
for (let i = 1; i<=10; i++) {
    console.log (i)    
}

for (let j = 10; j >= 1; j--) {  
    console.log(j)
}
```